resources/scripts/ttx_diff.py

```diff
@@ -70,10 +74,20 @@ def ttx(font_file: Path):
         ttx_file.name,
         font_file.name,
     ]
-    run(cmd, font_file.parent, "ttx.log")
+    run(cmd, font_file.parent, "ttx.log", None)
     return ttx_file
 
+# generate a simple text repr for gpos for this font
+def simple_gpos_output(font_file: Path, out_path: Path):
+    temppath = font_file.parent / "gpos.txt"
+    cmd = ["cargo", "run", "-p", "layout-repr", "--", font_file.name, "--table", "gpos"]
+    run(cmd, font_file.parent, "gpos.log", temppath.name,)
```

Comment:
maybe instead of `temppath.name` you could pass an open file object to redirect stdout to, and also make sure that you close the file after you finish writing to it, before you do the copy (this is good practice in general, and windows is very sensitive file handles left open). 

```python
with open(temppath.name, "w") as tempfile:
    run(cmd, font_file.parent, "gpos.log", tempfile)
```

Reply:
I've reworked this all so that the normalizer tool can just take a `-o` option to write to a file, and we don't need to worry about this weird stdout redirection hack.